ci/azure-pipelines-docs.yml

```diff
@@ -35,72 +35,8 @@ stages:
                 tests/integration/fixtures/**
                 tests/test_sets/**
 
-  - stage: custom_checks
-    dependsOn: scope_check
-    pool:
-      vmImage: 'ubuntu-latest'
-    jobs:
-    - job: link_checker
-      condition: or(eq(stageDependencies.scope_check.changes.outputs['CheckDocsChanges.DocsChanged'], true), eq(variables.isDevelop, true), eq(variables.isManual, true))
-      steps:
-      - bash: python docs/checks/docs_link_checker.py -p docs/docusaurus/docs -r docs/docusaurus/docs -s docs --skip-external
```

Comment:
I don't see an equivalent of `docs_link_checker` in the github actions version. Do you need to add this or was this intentionally skipped?